README.md

```diff
@@ -45,144 +54,87 @@ pip install eqlpy
 
 You can find the latest version on the [Python Package Index (PyPI)](https://pypi.org/project/eqlpy).
 
-## Usage
+TODO: How to install EQL itself?
 
-### Importing the package
+## Usage with Django
 
-```python
-from eqlpy import (
-    EqlInt,
-    EqlBool,
-    EqlDate,
-    EqlFloat,
-    EqlText,
-    EqlJsonb,
-    EqlRow
-)
-```
+### Defining an encrypted field
 
-### EQL value classes
+Let's say you have a customer model with an encrypted `name` field.
+You can use the `EncryptedText` type when defining the field in the model:
 
-Each EQL value class inherits from the base `EqlValue` class and handles specific data types.
+```py
+from django.db import models
+from eqlpy.eqldjango import *
 
-#### EqlInt
+class Customer(models.Model):
+    name = EncryptedText(table="customers", column="name")
+```
 
-Handles integer values.
+The above customer model would create a database table like this:
 
-```python
-eql_int = EqlInt(42, 'users', 'age')
-db_format = eql_int.to_db_format()
-print(db_format)
+```sql
+CREATE TABLE myapp_customer (
+    "id" bigint NOT NULL PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    "name" cs_encrypted_v1 NOT NULL,
+);
 ```
 
-#### EqlBool
+[!NOTE]
+The column type in the database is `cs_encrypted_v1` which is encrypted type defined in [EQL](https://github.com/cipherstash/encrypt-query-language/blob/main/docs/reference/PAYLOAD.md). Internally EQL just uses JSON.
 
-Handles boolean values.
+`eqlpy` supports many different data types, including numbers, dates and JSON.
+See the [Reference](#) docs for more information.
 
-```python
-eql_bool = EqlBool(True, 'users', 'is_active')
-db_format = eql_bool.to_db_format()
-print(db_format)
-```
+TODO: Link to the reference docs.
 
-#### EqlDate
 
-Handles date values.
+### Inserting data
 
-```python
-from datetime import date
+`eqlpy` automatically encrypts values when saving to the database.
```

Comment:
Are we describing the future state? Otherwise we'll need to explain this is done via CipherStash Proxy.

Reply:
Hmm. Yeah good call. I might be getting ahead of myself.